config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
+  get '/users/:id/posts', to: 'posts#index'
+  get '/users/:id/posts/:id', to: 'posts#show'
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
```

Comment:
- [ ] [OPTIONAL] You did great adding the routes but I think it would be better  to use resources and get less code, have a look at this :point_down: 

```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```